src/StringHelper.php

```diff
@@ -153,6 +153,14 @@ public static function replaceSubstring(string $string, string $replacement, int
      */
     public static function startsWith(string $input, ?string $with): bool
```

Comment:
What's the purpose to support null value?

Reply:
WDYT about to create an issue with this question?

Reply:
https://github.com/yiisoft/strings/issues/42